src/Entity/Udi.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * Entity class to keep list of issued UDI's.
+ *
+ * @ORM\Entity(repositoryClass="App\Repository\UdiRepository")
+ *
+ * @UniqueEntity(
+ *     fields={"uniqueDataIdentifier"},
+ *     errorPath="uniqueDataIdentifier",
+ *     message="This UDI already exists"
+ * )
+ */
+class Udi extends Entity
+{
+    /**
+     * A string containing the UDI.
+     *
+     * @var string
+     *
+     * @ORM\Column(type="text", nullable=false, unique=true)
+     */
+    protected $uniqueDataIdentifier;
+
+    /**
+     * Constructor.
+     *
+     * @param string $uniqueDataIdentifier The UDI string.
+     *
+     * Sets to UDI identifier for the UDI entity.
+     */
+    public function __construct(string $uniqueDataIdentifier): void
```

Comment:
I don't think constructors are allowed to have a return type

Reply:
It doesn't seem to break, but it's also not needed.

Reply:
It broke! I changed it after I tested. Gonna fix.